### PR TITLE
Add management command for generating billing statements

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

**Comment by hodgestar** [comment]
We seem to start all of our command names with `go_` and I think I'd like to stick with that for consistency.

**Comment by justinvdm** [comment]
Didn't add the prefix since the other billing management command didn't have a prefix. Should it also live with the other commands instead of in `go/billing/management/commands` then?

File: go/billing/management/commands/tests/test_gen_statement.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        user = self.user_helper.get_django_user()
+        self.user_email = user.email
+
+        account_number = user.get_profile().user_account
+        self.account = Account.objects.get(account_number=account_number)
+
+        self.command = Command()
+        self.command.stdout = StringIO()
+        self.command.stderr = StringIO()
```
Comment (hodgestar) [inline go/billing/management/commands/tests/test_gen_statement.py]: `go.base.test.helpers` now has `CommandIO` for this and I recently learned that `call_command` is the correct way to record all of this. See `go.base.tests.test_go_system_stats` for an example.

**Comment by hodgestar** [comment]
Left a few small comments, otherwise looks good.

**Comment by justinvdm** [comment]
Ready for re-review. Did you see [my question](https://github.com/praekelt/vumi-go/pull/1083#issuecomment-65207795)?

**Comment by hodgestar** [comment]
I like having the prefix and the command being in the `go/billing/commands`. :)

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**